File: Day19/Solution.java
class Calculator implements AdvancedArithmetic{

    public int divisorSum(int n){
        int divisorSum=n;
        for(int i=1;i<n;i++)
        {
            if(n%i==0)
                divisorSum += i;
        }
        return divisorSum;
    }
} 

File: Day19/Solution.cpp
class Calculator: public AdvancedArithmetic{
    int sum =1;
    public:
        int divisorSum(int num){
            for (int i=2;i<=num;i++){
                if (num%i==0){
                    sum +=i;
                }
            }
            return sum;
        }
};

